x-pack/plugins/security/public/nav_control/nav_control_service.tsx

```diff
@@ -49,6 +67,13 @@ export class SecurityNavControlService {
         }
       }
     );
+
+    return {
+      getUserMenuLinks$: () => this.userMenuLinks$.pipe(takeUntil(this.stop$)),
```

Comment:
I think we're missing the corresponding call in this service's `stop` lifecycle method:

```ts
  this.stop$.next();
```

Reply:
Nice catch! I updated the PR.